Makefile

```
@@ -0,0 +1,39 @@
+.PHONY: install
+install: ## Install dependencies using pip and pyenv
+	@echo "🚀 Creating virtual environment using pyenv and uv"
+	@uv venv .venv
+	@uv run pre-commit install
+
+.PHONY: check
+check: ## Check code quality and dependencies
```

checkmake (0.2.2)

[warning] Target body for "check" exceeds allowed length of 5 (6).

(maxbodylength)